web-ui/src/main/resources/catalog/components/utility/UtilityDirective.js

```diff
@@ -2596,6 +2596,41 @@
     }
   ]);
 
+  module.directive("gnInspireUsageDetails", [
+    "$http",
+    function ($http) {
+      return {
+        restrict: "A",
+        replace: true,
+        scope: {
+          inspireApiUrl: "=gnInspireUsageDetails",
+          inspireApiKey: "=apiKey"
+        },
+        templateUrl: "../../catalog/components/utility/partials/inspireapiusage.html",
+        link: function (scope, element, attrs) {
+          scope.inspireApiUsage = undefined;
+          if (
+            scope.inspireApiUrl &&
+            scope.inspireApiUrl.length > 0 &&
+            scope.inspireApiKey &&
+            scope.inspireApiKey.length > 0
```

Comment:
Could be use `angular.isArray`?

Reply:
Both variables are strings. The check here is a not undefined or not empty string.